extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -77,6 +78,9 @@ export class AmpAnalytics extends AMP.BaseElement {
      */
     this.type_ = null;
 
+    /** @private {!boolean} */
+    this.isScoped_ = element.hasAttribute('scope');
```

Comment:
shall we give it a different name? `scope` is already used in `selectionMethod` which has a quite different meaning.
how about `<amp-analytics sandbox>`